tools/generator/Java.Interop.Tools.Generator.ObjectModel/GenBase.cs

```diff
@@ -366,6 +366,23 @@ public void FixupMethodOverrides (CodeGenerationOptions opt)
 			set => support.FullName = value;
 		}
 
+		(object Package, object Name, string Id) javaFullNameIdCache;
+
+		public string JavaFullNameId {
```

Comment:
It would be nice to add a comment to this property explaining what the format is and what uses it.  There are *so many* `*Name` properties that I would like to start documenting what each one is for.

Reply:
doc-comment added.